src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: left 19px top 32px;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
```

Comment:
The selector '&:hover &__title' is incorrect. The '&' in SCSS refers to the parent selector, so this will not apply the hover effect as intended. You should use '.card:hover .card__title' instead to change the title color on hover.